content/en/setup/module-1/user-flows.mdx

```diff
@@ -57,9 +55,14 @@ If you would like to create dynamic pricing you will need to purchase Ocean toke
 
 Anyone can add liquidity to the pool you have created — they are known as liquidity providers. Liquidity providers earn a share of the transaction fees when the datatoken is purchased.
 
+### Free Pricing
+
+Ocean also supports assets to be published freely. Free pricing is suitable for individuals and organizations working in the public domain and want their datasets to be freely available. Publishers can also choose this model if they publish assets with licenses that require them to make them freely available.
+
+The page on [Pricing models](https://docs.oceanprotocol.com/concepts/asset-pricing/) explains more about the above mentioned asset pricing schemas.
 
 ## How does data consumption work in Ocean Protocol?
 
 - Go to Ocean Marketplace https://market.oceanprotocol.com/
 - Search for the data asset. The Ocean Marketplace provides features to search the Data/Algorithms by text, and users can also sort the result by published date.
-- Below image shows how to buy a Data asset with static content.
\ No newline at end of file
+- Go to [Download tutorial](https://docs.oceanprotocol.com/tutorials/marketplace-download-data-asset/) to learn how Ocean Market facilitates buying and downloading datasets.
```

Comment:
I like this approach of sending people over to the docs for more detail information and explanations, it helps avoid duplicating things. Could you add more links in to the docs anywhere it seems appropriate? For example, in the section on publishing, we can say "Go to [Publish tutorial]( www.docs.page... ) for a detailed guide on how to publish with Ocean Market"

Reply:
Added references to Architecture diagram, Publish tutorial, Fees page